tests/test_routes.py

```diff
@@ -1,8 +1,16 @@
 import json
+import os
 import pytest
 
 
-def test_classify(app):
+def test_classify(app, fs):
+    base_classifier_path = "./testdata"
```

Comment:
this is used in several places (e.g. flask_test_cfg), why not use from there?

Reply:
true, done